blocks/agent-communities/agent-communities.js

```diff
@@ -0,0 +1,105 @@
+import {
+  a, div,
+  img,
+  span,
+  p,
+} from '../../scripts/dom-helpers.js';
+import { getLoader } from '../../scripts/util.js';
+
+export default async function decorate(block) {
+  block.classList.add('cards', 'loading');
+  block.appendChild(getLoader('agent-communities'));
+
+  window.setTimeout(async () => {
+    // const url = `/commonwealth-real-estate-ma312${window.location.pathname}`;
+    const url = '/commonwealth-real-estate-ma312/east-greenwich/barry-alofsin/cid-473521';
```

Comment:
What should this URL be?

Reply:
The commented one

Reply:
Are we sure this `commonwealth-real-estate-ma312` is correct, as well?

Reply:
Yes, this is the Api which is currently called for fetching communities.

Reply:
Yes, this is the Api which is currently called for fetching communities.